deepchecks/nlp/checks/data_integrity/under_annotated_segments.py

```diff
@@ -30,15 +30,20 @@
 __all__ = ['UnderAnnotatedMetaDataSegments', 'UnderAnnotatedPropertySegments']
 
 MAX_SAMPLES_IN_FIGURE = 1000
+# The threshold the UnderAnnotatedSegments considers the data to be well
+# annotated and skips the checks
+ANNOTATION_RATIO_THRESHOLD = 95.0
+MIN_TEXT_SAMPLES = 10  # Min samples to calculate under annotated segments
 
 
 class UnderAnnotatedSegments(SingleDatasetCheck, WeakSegmentAbstract):
     """Check for under annotated data segments."""
 
     def __init__(self, segment_by: str, columns: Union[Hashable, List[Hashable], None],
                  ignore_columns: Union[Hashable, List[Hashable], None], n_top_features: int,
-                 segment_minimum_size_ratio: float, n_samples: int,
-                 categorical_aggregation_threshold: float, n_to_show: int, **kwargs):
+                 segment_minimum_size_ratio: float, n_samples: int,  n_to_show: int,
+                 categorical_aggregation_threshold: float, annotation_ratio_threshold: float,
```

Comment:
annotation_ratio_threshold should be a float

Reply:
Made it a const instead of parameter